fbw-a32nx/src/systems/fmgc/src/guidance/lnav/legs/HX.ts

```diff
@@ -143,36 +144,31 @@ abstract class HXLeg extends XFLeg {
         const radius = this.radius;
         const turnSign = this.turnDirection === TurnDirection.Left ? -1 : 1;
 
-        const fixA = Avionics.Utils.bearingDistanceToCoordinates(
+        const fixA = placeBearingDistance(
```

Comment:
I'd rather stick with Avionics.Utils here for now, and move to ECEF after fms-v2 merges. The msfs-geo functions have known issues that cause path glitches in other leg types and I'd rather not start having such issues with holds.